warehouse/accounts/services.py

```diff
@@ -10,18 +10,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import base64
 import collections
+import enum
```

Comment:
Unused?

Reply:
Hm, stange, when I submitted it, flake8 was happy. Probably a merge issue. I’ll make flake8 happy again.

Reply:
Ok found.

This line (predating the PR) actually disables flake8 for the whole file
https://github.com/pypa/warehouse/blob/696162baf8a0efc140b487ac888c8458fa8a9b9b/warehouse/accounts/services.py#L109

Reply:
Ah, that looks like an issue with our auto formatter moving the comment, it should be on the following line(s).

Reply:
No this comment always acts on the whole file: https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#ignoring-entire-files

I've made a PR: https://github.com/pypa/warehouse/pull/8317

Reply:
The regex is sused to mach any line in the file, and ignore the whole file.

https://github.com/PyCQA/flake8/blob/076dfeee372affb1dbe849d12a1cc20f8ea37d40/src/flake8/defaults.py#L43
https://github.com/PyCQA/flake8/blob/94304de68480ac3007377404837130df72534ee3/src/flake8/processor.py#L386-L388